src/app/zap-templates/templates/app/CHIPClientCallbacks.zapt

```diff
@@ -35,3 +36,16 @@ typedef void (*{{asCamelCased parent.name false}}Cluster{{asCamelCased name fals
 {{/if}}
 {{/chip_client_clusters}}
 {{/if}}
+
+// List specific responses
+{{#chip_client_clusters}}
+{{#chip_server_cluster_attributes}}
+{{#if isList}}
+{{#if isStruct}}
+typedef void (*{{asCamelCased parent.name false}}{{asCamelCased name false}}ListAttributeCallback)(void * context, uint16_t count, _{{type}} * entries);
+{{else}}
+typedef void (*{{asCamelCased parent.name false}}{{asCamelCased name false}}ListAttributeCallback)(void * context, uint16_t count, {{chipType}} * entries);
```

Comment:
Again, document the type bits here.

Reply:
In order to make it easier, I changed the templates so we always use `chipType`